src/objecttypes/config/demo.py

```diff
@@ -0,0 +1,60 @@
+from django.conf import settings
+from django.urls import reverse
+
+import requests
+from django_setup_configuration.configuration import BaseConfigurationStep
+from django_setup_configuration.exceptions import SelfTestFailed
+
+from objecttypes.token.models import TokenAuth
+from objecttypes.utils import build_absolute_url
+
+
+class DemoUserStep(BaseConfigurationStep):
+    """
+    Create demo user to request Objectypes API
+    """
+
+    verbose_name = "Demo User Configuration"
+    required_settings = [
+        "DEMO_TOKEN",
+        "DEMO_PERSON",
+        "DEMO_EMAIL",
+    ]
+    enable_setting = "DEMO_CONFIG_ENABLE"
+
+    def is_configured(self) -> bool:
+        return TokenAuth.objects.filter(token=settings.DEMO_TOKEN).exists()
+
+    def configure(self):
+        token_auth, created = TokenAuth.objects.get_or_create(
```

Comment:
We could maybe use `update_or_create` here https://docs.djangoproject.com/en/4.2/ref/models/querysets/#update-or-create, that way we can remove the if block below